Main Model/household-procedures.nls

```diff
@@ -49,7 +49,6 @@ to search-for-employement
   ] [
     search-better-paid-job
   ]
-
 end
 
 ;household proceudre, searches for a potential employer under the condition that this household is unemployed, a more rigorous search than "search better paid job"
```

Comment:
In search-better-paid-job-logic, get rid of the if statement. That is left over from Lengnick. We do want to let workers quit even if they are last worker.